digitaltwins/device/test/_digital_twin_client_test.js

```diff
@@ -374,140 +301,72 @@ describe('Digital Twin Client', function () {
       });
     });
 
-    const timeoutForDisconnects = 30;
-    const almostToTimeout = 29000;
-    const pushOverTheTimeout = 2000;
-    describe('Will timeout in ' + timeoutForDisconnects + ' seconds if the initial registration message disconnects the connection', () => {
-      let dtClient;
-      let registrationDeviceClient;
-      let fakeInterfaceInstance;
-
+    /* Tests_SRS_NODE_DIGITAL_TWIN_DEVICE_41_XXX: [] */
+    it('will enable methods for all commands in all interfaceInstances', function (done) {
+      const registrationDeviceClient = {
```

Comment:
This is actually the first test that will run for the propertyupdates.  Need to change the comment will enable methods for all commands.